draco/analysis/flagging.py

```diff
@@ -1214,7 +1224,9 @@ def mask_multi_channel(self, power, mask, times):
         )
 
         # Mask bright data with bright sources and daytime removed
-        summask = rfi.sumthreshold(power, start_flag=mask, max_m=self.max_m)
+        summask = rfi.sumthreshold(
```

Comment:
Will the threshold be in the correct units to compare to the power here?  I think power needs to be divided by some estimate of the noise to put it into units of sigma.  Or we could pass an estimate of the noise into sumthreshold with the variance argument.